lib/components/link.js

```diff
@@ -89,12 +89,16 @@ function computeTag(props) {
     return props.to && !props.disabled ? "router-link" : "a";
```

Comment:
Maybe this should check for `parent.$router` as well?

Reply:
Yes i think it should too.

Reply:
Just need to make sure its pass `parent` as the next arg